resource_customizations/kubevirt.io/VirtualMachine/health.lua

```diff
@@ -0,0 +1,15 @@
+hs = { status="Progressing", message="No status available"}
+if obj.status ~= nil then
+  if obj.status.created ~= nil then
+    message="VMI Created"
```

Comment:
I guess you wanted to set `hs.message` here?

Reply:
Oops, thanks, I just force-pushed a quick update for this.